showcase/src/app/shared/components/example-viewer/example-viewer.component.html

```diff
@@ -1,7 +1,10 @@
-<mat-card class="stark-card">
+<mat-card [id]="id" class="stark-card">
 	<mat-card-header>
-		<mat-card-title>
-			<h3 translate>{{ exampleTitle }}</h3>
+		<mat-card-title fxLayout fxLayoutAlign="start center">
+			<a *ngIf="id" mat-icon-button [uiSref]="exampleState" [uiParams]="{ '#': id }">
```

Comment:
Can you add a unit test for this new functionality?  Also when there is no `id` set?